analyzers/tests/SonarAnalyzer.UnitTest/MetadataReferences/MetadataReferenceFacade.cs

```diff
@@ -87,6 +87,17 @@ internal static class MetadataReferenceFacade
             };
 #endif
 
+        internal static References GetSystemDiagnosticsProcess() =>
```

Comment:
Because `System.DiagnosticsProcess` doesn't work without it. Line
```
private Process field = Process.Start("file.exe");
```
fails with

>The type 'Component' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.ComponentModel.Primitives, Version=5.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a

because type `Process` inherits `Component`

Reply:
could you add a comment specifying this, please? it will help future maintainers